Dockerfile

```diff
@@ -7,6 +7,10 @@
 FROM tomcat:8-jre8
 MAINTAINER Fedde Schaeffer <fedde@thehyve.nl>
 
+ARG BRANCH=${BRANCH:-v1.11.2}
+ARG REPOSITORY=https://github.com/cBioPortal/cbioportal.git
```

Comment:
we can change default by using `master` branch

Reply:
I apologise if I'm misreading the syntax, but wouldn't overriding the default also be possible if the line were to say just the following?
```Dockerfile
ARG BRANCH=v1.11.2
```

The way I read [this documentation](https://docs.docker.com/engine/reference/builder/#default-values), that would suffice to specify a default in case no `--build-arg` param is passed to `docker build`.